contrib/test/requirements.txt

```diff
@@ -1,54 +0,0 @@
-# Pip requirements file for Ansible-based integration-testing environment.
```

Comment:
It cost me time to realize that this was unrelated. Sometimes incidental cleanup can be a good part of a commit, but in this case it looks completely random; something that belongs in a separate cleanup PR.

Reply:
At least I made a separate commit for it :smile: Yes, it could be moved.

Reply:
Okay, I'll move it.